### PR TITLE
Pair programming coding display menu on click

### PR DESCRIPTION
**Pull request by fredo509**
**Portfolio - mobile menu**

** Description **
For this milestone of my portfolio website, I will implement the mobile menu using your recently acquired JavaScript knowledge.

** General requirements **

- Make sure that there are [no linter errors].
- Make sure that you used correct [GitHub flow].
- Make sure that you documented your work [in a professional way].

** HTML/CSS & JavaScript requirements **
Follow [best practices for HTML & CSS]
Follow  [best practices for JavaScript]

**Project requirements**

- For this project, you should add the [JavaScript + HTML + CSS linter].

In mobile, implement the following features:

- When the user clicks (or taps) the hamburger button, the mobile menu appears.
- When the user clicks (or taps) the close (X) button, the mobile menu disappears.
- When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
- When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+const iconMenu = document.querySelector('#ml');
+const menu = document.getElementById('it-nav');
+iconMenu.addEventListener('click', () => {
+  menu.classList.toggle('showMenu');
+  iconMenu.classList.toggle('changeIcon');
+});
```
Comment (Mihndim2020) [inline script.js:7]: - [ ] As it is a [requirement ](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md) for this project, "`When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.`", please make sure the mobile menu disappears when a user selects of tap on a particular option. After selecting an option, the mobile menu doesn't disappear, please kindly fix that.

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
### Hi Team, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Clean and Organized code 
:heavy_check_mark: Descriptive Readme file

There are some issues that you still need to work on to get this project approved, but you are almost there!
ALMOST-MINION

# Status: Requires Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can get to me on slack  using this handle @mihndim.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js (hunk 1 above)
Comment (huseyinbiyik) [reply script.js:7]: As the previous reviewer requested, please make sure after clicking on any link, the nav closes.

Screen Recording 2023-03-08 at 1 32 19 AM

**Review by huseyinbiyik — CHANGES_REQUESTED** [review]
Hi team,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
tenor

 ## Highlights
- The text animation on h1 looks fantastic  👌 
- Linter set up well and checks pass 🟢
- Github flow followed 👌
- .gitignore used to ignore requested files 📁
- Commit messages are meaningful and written in imperative tense 📝
- PR description is descriptive 📝


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
⚠️  Kindly follow Github flow. You are asking Pull Request to merge from feature branch to feature branch.
Screenshot 2023-03-08 at 1 40 30 AM

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Shedrack-Sunday — APPROVED** [review]
Hi @Booze33 @fredo509 

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

image


# Highlights
- No linter errors ✔️
- Good implementations ✔️


## Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

__check the comments for details

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**